### ISSUE TITLE
Cannot start a Self-Hosting

### ISSUE THREAD

**Issue by mariohik** [exceptionless/Exceptionless#607]
The 'docker-compose up' command returns the following error:
```
/usr/share/dotnet/sdk/3.1.201/NuGet.targets(124,5): error: Unable to load the service index for source https://f.feedz.io/foundatio/foundatio/nuget/index. json. [/app/Exceptionless.sln]
/usr/share/dotnet/sdk/3.1.201/NuGet.targets(124,5): error: Resource temporarily unavailable [/app/Exceptionless.sln]
ERROR: Service 'api' failed to build: The command '/bin /sh -c dotnet restore' returned a non-zero code: 1
```

**Comment by niemyjski** [comment]
It sounds like the cdn was down to restore packages, please try again.

**Comment by mariohik** [comment]
No, it's not.
The error persists.

**Comment by niemyjski** [comment]
Can you view: https://f.feedz.io/foundatio/foundatio/nuget/index.json

**Comment by mariohik** [comment]
Yes

**Comment by Amber-Wish** [comment]
Hi guys! I encountered the same problem. How can I solve it?

**Comment by alexanderadam** [comment]
> It sounds like the cdn was down to restore packages, please try again.

Is the application required to work online? I thought Docker images should usually contain everything needed?

**Comment by ejsmith** [comment]
I think the problem people are running into is that the root [docker-compose.yml](https://github.com/exceptionless/Exceptionless/blob/master/docker-compose.yml) is also doing a code build. We need to have some separate ones that use pre-built images from Docker Hub.

**Comment by ejsmith** [comment]
@alexanderadam I just updated that file to not do a code build and also updated the self-hosted docs. Can you try again?

**Comment by alexanderadam** [comment]
I didn't try it yet, I was just curious because I saw this issue. But @mariohik might give some feedback.

**Comment by mariohik** [comment]
As I don't have a lot of knowledge of Linux, I solved it in another way. What I did was as follows:
1 - I installed Docker for Windows on my notebook
2 - I generated the images using the 'docker-compose up' command on it
3 - I exported the images with the command 'docker save <images> -o exceptionless.tar'
4 - I imported the images on the desired computer using 'docker load -i exceptionless.tar'
5 - I renamed the images that were not with the correct name 'docker image tag d583c3ac45fd exceptionless/api: latest'
6 - Then I ran the command 'docker-compose up' normally.

I know it's not the ideal solution, but it worked.
